baggage/README.md

```diff
@@ -1,10 +1,26 @@
 # Baggage Header
 
-`baggage` header is used to propagate properties not defined in `traceparent`. There
-are two common use cases. First is to define a context on trace initiation. Such context will
-have customer's identity, high-level operation name and other properties like a flight name.
-Second use case is to pass the caller's name to the next component. This name-value pair will be
-overridden by every service to it's own name.
+Applications and libraries can use baggage to attach user-defined properties as part of a distributed request. A framework can automatically propagate this data to downstream services as part of the distributed context. The benefit of the cross-cutting nature of this context propagation means that it doesn't require any changes in the participating services (e.g., to change API signatures) to propagate these parameters. There are two main categories of use cases of baggage: 
+
+1. Use cases that enable better _observability_ of the system.
+2. Use cases that enable better _control_ of the system.
+
+## Example use cases
+The below are a few example use cases for baggage:
+
+### Labeling synthetic traffic
+Baggage can be used to tag synthetic requests (e.g., isProduction=false). This can help services distinguish between production requests and synthetic requests. For example, this can be used by a system to partition error rate metrics to have separate time series for production and synthetics. This can enable configuring alert thresholds independently for production and synthetics.
+
+In addition, services can use baggage to customize behavior for synthetic requests. For example, a service can choose to redirect synthetic requests to a different cluster.
+
+### Attributing resource usage to lines of business
+Baggage can be used to attribute infrastructure spend to a line of business. While higher level services will likely have dedicated services for each line of business, lower layers such as storage or messaging layers might be shared across multiple lines of businesses. A client or an upstream service can use baggage to attach information (e.g., lob=youtube) on the line of business associated with that request. Downstream services can use this to attribute operations (e.g., number of storage operations) to a specific line of business.
+
+### Traffic prioritization/QoS
+Baggage can be used by services to prioritize requests to provide better QoS. In a system, there can be different types of requests with different priorities: e.g., in ride sharing app, request for making a trip is more important than adding a location to favorites. As this request propagates through the system, this information can be lost in the lower shared layers. To solve this, we can assign tiers to different requests and pass it as part of baggage. In case of a traffic spike, the services can prioritize based on it.
+
+### Additional labels/annotations for Observability
```

Comment:
This one is very vague. Can we convert it into a concrete example? What specific information are you thinking of?

Reply:
The use case I had in mind was an Observability framework using the baggage information to create dimensions/labels in a metrics system. Agree this is a bit vague and I think this is already covered in the Observability related use cases above (e.g., multiple time series for error metrics for the synthetics vs. production traffic). I will remove this as a use case, however I think it will be good to retain the second part of this - the info about the difference between tracestate and baggage - will move that to earlier in this document before the use cases section.

Reply:
Yes, if it's just a general principle of partitioning metrics with baggage dimensions then it's not really a concrete use case, I would remove it unless there's some specific example different from synthetic traffic example.

Reply:
Thanks Yuri, that sounds good, I have removed this section. Added two more use cases (chaos engineering and manual testing) per your feedback.